gauge.go

```diff
@@ -103,6 +107,9 @@ func (g *Gauge) Buffer() []Point {
 				p.Bg |= AttrReverse
 			}
 
+			if g.PercentColorHighlighted != Attribute(uint16max) {
```

Comment:
While I think this setting can be used by other widgets in the same manner, since other widgets probably also need some kind flag to provide the same functionality. Or `ColorReset` is better?